src/population_loader.rs

```diff
@@ -0,0 +1,44 @@
+use crate::parameters_loader::Parameters;
+use ixa::{
+    context::Context,
+    define_derived_property, define_person_property, define_person_property_with_default,
+    global_properties::ContextGlobalPropertiesExt,
+    people::{ContextPeopleExt, PersonId},
+};
+
+use serde::Deserialize;
+
+#[derive(Deserialize, Debug)]
+#[allow(non_snake_case)]
+pub struct PeopleRecord {
+    age: u8,
+    homeId: usize,
+}
+
+define_person_property!(Age, u8);
+define_person_property!(HomeId, usize);
+define_person_property_with_default!(Alive, bool, true);
+
+define_derived_property!(CensusTract, usize, [HomeId], |home_id| { home_id / 10000 });
+
+pub fn create_person_from_record(context: &mut Context, person_record: &PeopleRecord) -> PersonId {
+    context
+        .add_person(((Age, person_record.age), (HomeId, person_record.homeId)))
+        .unwrap()
```

Comment:
Same comment about `expect` over `unwrap` here. Whether you want this to propagate an error is up to you -- I think it should, especially if we will end up using this function to potentially create people via births in the future (so the function's signature would be `Result<PersonId, IxaError>`). But, I can see an argument where you just want to use `expect` here since we've already validated the data down below in the `init` function.

Reply:
agree with ixaerror being the right way to go here